template/README.md

```diff
@@ -182,20 +202,39 @@ vagrant ssh -c 'vault read consul/creds/dev-team'
 vagrant ssh -c 'vault read consul/creds/admin-team'
 ```
 
-*Tokens can be used to access UI (different access level depends on role)
+> :bulb: Tokens can be used to access UI (different access level depends on role)
+
+
+## Usage
+### Commands
+There are several commands that help to run the vagrant-box:
+- `make install` installs all prerequisites
+
+- `make up` provisions a [vagrant-hashistack](https://github.com/fredrikhgrelland/vagrant-hashistack/) box on your machine. After the machine and hashistack are set up it will run the [Default Startup Scheme](#default-startup-scheme).
+
+- `make clean` takes down the provisioned box if there is any.
+
+- `make update` downloads the newest version of the [vagrant-hashistack box](https://github.com/fredrikhgrelland/vagrant-hashistack/) from [vagrantcloud](https://vagrantcloud.com/fredrikhgrelland/hashistack).
+
+- `make example` runs the example in [template_example/](template_example)
+
+> :bulb: For full info, check `template/Makefile`.
+> :warning: Makefile commands are not idempotent in the context of vagrant-box.  You could face the error of port collisions. Most of the cases it could happen because of the vagrant box has already been running.
```

Comment:
We need to specify the remedy as well to get around this problem/situation...like run ```make clean``` or run 
```
vagrant global-status
vagrant destroy -f <id> 
```

Reply:
Fixed in latest commit, thanks!